os/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e/us_ticker.c

```diff
@@ -57,15 +57,16 @@ void us_ticker_init(void)
 }
 
 #if (!TICK_READ_FROM_CPU) || !defined(PLATFORM_FREERTOS)
-uint32_t us_ticker_read(void)
+uint32_t us_ticker_read(void) 
 {
-	uint32_t tick_cnt;
+	uint32_t tick_cnt, us_tick_32;
 	uint64_t us_tick;
 
 	tick_cnt = SYSTIMER_TickGet(); //up counter
 	us_tick = tick_cnt * (1000000 / 32768);
+	us_tick_32 = us_tick & 0xFFFFFFFF;
```

Comment:
@fengyunlai As I left, I will follow your decision. But this pr will not resolve the SVACE issue. Because line 66 has already tainted value due to type mismatching.
`tick_cnt * (.... )` has uint32_t value, not uint64_t value. This line is unnecessary and can't resolve the issue. Could you check it again?